frontend/src/components/events/molecules/event_card.css

```diff
@@ -0,0 +1,10 @@
+.eventCard {
+  /* border: 0.5px solid gray; */
+  border-radius: 5px;
+  transition: box-shadow 0.3s;
+  max-width: 400px;
```

Comment:
i think you can just remove this -> flex handles it fine

Reply:
When I remove the max-width the text stretches out the card
![without-max](https://user-images.githubusercontent.com/36250779/64747373-70139e80-d4d4-11e9-8761-83d96372f983.png)

I will continue to play around with some of the other css styling, but by leaving the max-width it keeps the styling looking very concise.
![with-max](https://user-images.githubusercontent.com/36250779/64747424-9d604c80-d4d4-11e9-80c3-cf8b5c1de465.png)